components/applications-service/pkg/storage/client.go

```diff
@@ -58,6 +88,7 @@ type Service struct {
 	Channel             string
 	Site                string
 	PreviousHealth      string    `db:"previous_health"`
+	UpdateStrategy      string    `db:"update_strategy"`
```

Comment:
I'm a little surprised about db struct tags here; in `pkg/storage/postgres`, I understand them. Here? Not really.

Reply:
yeah, I might have to fix that in a next PR. I agree with what you are mentioning here!

Reply:
Another fix that you pointed out @srenatus https://github.com/chef/automate/pull/906